xtable-core/src/main/java/org/apache/xtable/catalog/Constants.java

```diff
@@ -24,8 +24,26 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class Constants {
 
+  /**
+   * This property should be used to specify the data source provider that Spark uses to read from
+   * or write to the data source. For ex: when working with a Delta table in AWS Glue, this property
+   * typically points to the Delta Lake format
```

Comment:
When you say `format` do you mean the `_delta_log` directory?

Reply:
Can you include the expected values in the comment? It may be more clear. With "points to" I was thinking of a pointer to a location on the first pass.